ports/stm32/powerctrl.c

```diff
@@ -492,6 +502,11 @@ void powerctrl_enter_stop_mode(void) {
     // executed until after the clocks are reconfigured
     uint32_t irq_state = disable_irq();
 
+    // Disable SysTick Interrupt
+    // Note: This seems to be required at least on the H7 REV Y,
+    // otherwise the MCU will leave stop mode immediately on entry.
+    SysTick->CTRL &= ~SysTick_CTRL_TICKINT_Msk;
```

Comment:
This is strange.... it means that the SysTick IRQ can now wake from STOP mode.  Is there anything in the datasheet about this?

Reply:
No not that I could find, I searched for a while and checked the latest errata, I think it may be an undocumented silicon bug because this does Not happen on H743 REV-V. SysTick IRQ gets disabled with the other IRQs as you can see from the gdb log, but then triggers on entry to STOP mode. Disabling it like that shouldn't have any side effect on other MCUs though.

Reply:
Actually after thinking about it some more, it may not be disabled at all because when single stepping with the debugger systick counter is suspended as far as I know.

Reply:
Ok.  But I'd say this should be guarded by `defined(STM32H7)` because this cause is not properly understood.

Reply:
Done.